src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -300,48 +356,78 @@ export default class StripViewModel extends BaseModel {
      *
      * @for StripViewModel
      * @method _render
+     * @chainable
      */
     _redraw() {
         return this;
     }
 
-
     /**
+     * Disable the instance ane tear down handlers
      *
-     *
-     * @method reset
+     * @for StripViewModel
+     * @method disable
      */
-    reset() {
+    disable() {
         this.$element.off('click', this.onClickHandler);
         this.$element.off('dblclick', this.onDoubleClickHandler);
+
+        return this;
     }
 
     /**
-     *
+     * Destroy the instance
      *
      * @for StripViewModel
-     * @method shouldUpdate
-     * @param  aircraftModel {AircraftInstanceModel}
-     * @return {boolean}
+     * @method destroy
+     * @chainable
      */
-    shouldUpdate(aircraftModel) {
-        return aircraftModel.fms.currentPhase !== this._flightPhase ||
-            aircraftModel.mcp.altitude !== this._altitude ||
-            aircraftModel.mcp.speed !== this._speed;
+    destroy() {
+        this.disable();
+        this.$element.remove();
+
+        this.id = '';
+        this._eventBus = null;
+        this.$element = null;
+        this.aircraftId = '';
+        this._callsign = '';
+        this._aircraftType = '';
+        this._transponder = 1200;
+        this._assignedAltitude = -1;
+        this._flightPlanAltitude = -1;
+        this._arrivalAirport = '';
+        this._departureAirport = '';
+        this._alternateAirport = '';
+        this._flightPlan = '';
+        this.$callsignView = null;
+        this.$aircraftTypeView = null;
+        this.$transponderView = null;
+        this.$flightPlanAltitudeView = null;
+        this.$arrivalAirportView = null;
+        this.$departureAirportView = null;
+        this.$alternateAirportView = null;
+        this.$flightPlanView = null;
```

Comment:
Oh, okay I see. Maybe a comment to that effect then. Anything other than alphabetical really should have an explanation so people who come across it (like me) don't mess up the convention because they don't pick up on the pattern.